charts/prod-ftvalabdata-values.yaml

```diff
@@ -49,6 +49,12 @@ django:
     db_user: "ftva-digital-data"
     db_host: "p-d-postgres.library.ucla.edu"
     db_port: 5432
+    email_backend: "django.core.mail.backends.smtp.EmailBackend"
```

Comment:
Since this is explicitly set in `settings.py`, no need for a variable here.  Being explicit in `settings.py` means the environment can't override it.